src/main/webapp/js/statusMessage.js

```diff
@@ -1,7 +1,13 @@
 $(document).ready(function() {
     var statusMessage = document.getElementById('statusMessage');
-    var navbarHeight = document.getElementsByClassName('navbar')[0].offsetHeight;
+    var navbarHeight = 0;
```

Comment:
could name it as navbarOffsetHeight but it's really not that important if we're trying to release it asap.